.github/ISSUE_TEMPLATE/migration_task

```diff
@@ -0,0 +1 @@
+Please add screenshots, links to code, any relevant information, and then add any relevant tag, e.g.:  "Test backfill" for when more tests are needed, or "you can do this" for things that don't need any extra context and are good issues for new folks to help out with.
```

Comment:
should this file's extension be `.md`?
Also, perhaps line breaks would make it easier to read in editors that don't break lines automatically?

Reply:
YESSSSSSSS

Reply:
✅